NuGet.config

```diff
@@ -12,6 +12,7 @@
       one as a template. The following line is a marker to insert the test restore sources.
     -->
     <!-- TEST_RESTORE_SOURCES_INSERTION_LINE -->
+    <add key="local" value="./packages" />
```

Comment:
You probably need to update this to point to a public feed that you own, i.e. myget and publish the locally built Microsoft.Net.Compilers.Toolset package there. That's what I usually do when I need to validate my changes with CI.